lib/validate.ts

```diff
@@ -289,8 +291,10 @@ function getValidationSchemaSource(version: string): Schema | null {
         return componentsDefinitionSchema_v1_9_x;
     } else if (semver.satisfies(version, '1.10.0', semVerOptions)) {
         return componentsDefinitionSchema_v1_10_x;
-    } else if (semver.satisfies(version, '1.11.0-next', semVerOptions)) {
+    } else if (semver.satisfies(version, '1.11.0', semVerOptions)) {
```

Comment:
This is not how the versioning works for `studio-component-set-tools`. The tables feature will be part 1.11 (which is the version under development at the moment). See https://github.com/WoodWing/studio-component-set-tools/blob/f2be3159eeff8cb0d4f607edd0aac9fbacc9f744/README.md#releasing-a-new-schema-version.

Also, this kind of change needs to be in a separate PR.